src/qappfw.js

```diff
@@ -208,32 +208,24 @@ class QRadar {
      *
      * @returns {Object} The currently logged in QRadar user.
      *                   WARNING this function uses a synchronous JavaScript call.
-     *
      */
-    static getCurrentUser()
-    {
+    static getCurrentUser() {
         let currentUser = null;
+        const params = "current_user=true";
```

Comment:
Why not just hardcode this into the path below?

Reply:
will change to hardcode it was just thinking if that changed but then you'd probably just be changing one line anyway